doc/classes/Label3D.xml

```diff
@@ -32,6 +32,12 @@
 		</method>
 	</methods>
 	<members>
+		<member name="alpha_antialiasing_edge" type="float" setter="set_alpha_antialiasing_edge" getter="get_alpha_antialiasing_edge" default="0.0">
+			Threshold at which antialiasing will be applied on the alpha channel.
+		</member>
+		<member name="alpha_antialiasing_mode" type="int" setter="set_alpha_antialiasing" getter="get_alpha_antialiasing" enum="BaseMaterial3D.AlphaAntiAliasing" default="0">
+			The type of alpha antialiasing to apply. See [enum BaseMaterial3D.AlphaAntiAliasing].
+		</member>
 		<member name="alpha_cut" type="int" setter="set_alpha_cut_mode" getter="get_alpha_cut_mode" enum="Label3D.AlphaCutMode" default="0">
 			The alpha cutting mode to use for the sprite. See [enum AlphaCutMode] for possible values.
 		</member>
```

Comment:
Two interesting inconsistencies here:
- `alpha_antialiasing_mode` but the setter/getter doesn't include `_mode` suffix
- `alpha_cut` but the setter/getter includes `_mode` suffix

Then the enum names differ too on their use of `Mode` or not.

I'm not sure we're particularly consistent with this in the codebase but which should probably aim to be within a given class at least.

Reply:
> `alpha_antialiasing_mode`

It's the same as in `BaseMaterial3D`.

I'm not sure if it is worth making breaking changes to `alpha_cut` or `BaseMaterial3D` properties.